inputsource.cpp

```diff
@@ -181,6 +195,82 @@ void InputSource::cleanup()
     }
 }
 
+void InputSource::readMetaData(const char *filename)
+{
+    QString dataFilename;
+
+    std::unique_ptr<QFile> dataFile(new QFile(dataFilename));
+    QFile datafile(filename);
```

Comment:
Dead code? Looks like the unique_ptr version should be removed.

Reply:
A bug in fact (`dataFile` was erroneously used to get an error message later on)